webplugin/js/app/km-rich-text-event-handler.js

```diff
@@ -469,8 +469,10 @@ Kommunicate.richMsgEventHandler = {
         console.log("passenger detail submitted");
     },
     processQuickReplies : function(e){
-       var message = e.target.title;
-       var metadata = {};
+        var message = e.target.title;
+        var metadata = {
```

Comment:
@akshatdhabi is this going to be sent to server? if yes, then need to find some other solution, need to figure out a way to achieve this on client side itself